cypress/support/index.d.ts

```diff
@@ -559,8 +559,9 @@ declare namespace Cypress {
      * Expect the table to have a specific number of rows
      *
      * @param numberOfRows - the number of rows
+     * @param modal - optional, default = false - use true if the field is in a modal overlay
```

Comment:
'....use true is the **field** is in a modal overlay'...it's a bit misleading...which field? You mean the table for which we count the number of rows? Then I would suggest '..use true if the table is in a modal...' or better 'component'